man/systemd.socket.xml

```diff
@@ -216,6 +216,11 @@
         <varname>BindIPv6Only=</varname> setting (see below).
         </para>
 
+        <para>If the address string is a string in the format
+        vsock:x:y, it is read as CID x on a port y address in the
+        <constant>AF_VSOCK</constant> family.  Specifying the CID is
+        optional so vsock::y is accepted as well.</para>
```

Comment:
Again, `<literal></literal>`...

the english sounds a bit off here. Maybe just say "Specifying the CID is optional, and may be set to the empty string in that case" or so?